### PR TITLE
docs: update `README.md` with newly introduced icons

### LINKED ISSUE #829: [DOCS] docs - update icons in readme
### Description

As prime icons got replaced with react icons,
Some new icons got added as well .
But docs are  still not updated yet , These are the newly added icons :

1.instagram   
2.linkedin 
3.twitch
4.dev.to
5.codewars
6.hashnode
7.graduation-hat

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by RanjeetNSuthar**
## Fixes Issue

Closes #829 

## Changes proposed

Following is the list of newly added icons:
1. Instagram
2. Linkedin
3. twitch
4. dev.to
5. Codewars
6. Hashnode
7. Graduation-hat
8. Send
9. Tiktok ( as suggested by @emmalearnscode )
10. Search
11. Left Arrow

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

Recently added Search and Arrow icons are also included

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/844"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -105,8 +105,12 @@ Your `avatar` URL should take the format of `https://github.com/<yourusername>.p
...
+| Book     | <img src="https://user-images.githubusercontent.com/76985777/145391108-f8c08f8e-679f-45a3-ad58-83ef60aa28fe.png" width=100% height=30%> |  Dollar   | <img src="https://user-images.githubusercontent.com/76985777/145393429-c03c5c3e-0416-4d28-be21-8a01a0c3dff5.png" width=100% height=30%> |   Instagram   | <img src="https://user-images.githubusercontent.com/76241195/147094225-202bb79d-5e06-4169-a0aa-712646f1159a.png" width=100% height=30%>  |
+| Linkedin     | <img src="https://user-images.githubusercontent.com/76241195/147094289-252d533d-aeae-493c-a21a-21538162cea6.png" width=100% height=30%> |  Twitch   | <img src="https://user-images.githubusercontent.com/76241195/147094354-12d3b38d-bb87-4c52-a3b7-4d934498da31.png" width=100% height=30%> |   dev.to   | <img src="https://user-images.githubusercontent.com/76241195/147094431-b194a188-0a63-4ce3-89c9-1d7578405318.png" width=100% height=30%>  |
```
Comment (Panquesito7) [inline README.md:110]:
```suggestion
| LinkedIn     | <img src="https://user-images.githubusercontent.com/76241195/147094289-252d533d-aeae-493c-a21a-21538162cea6.png" width=100% height=30%> |  Twitch   | <img src="https://user-images.githubusercontent.com/76241195/147094354-12d3b38d-bb87-4c52-a3b7-4d934498da31.png" width=100% height=30%> |   DEV.to   | <img src="https://user-images.githubusercontent.com/76241195/147094431-b194a188-0a63-4ce3-89c9-1d7578405318.png" width=100% height=30%>  |
```

File: README.md [hunk 2]
```diff
@@ -105,8 +105,12 @@ Your `avatar` URL should take the format of `https://github.com/<yourusername>.p
...
+| Hashnode     | <img src="https://user-images.githubusercontent.com/76241195/147095015-e009e115-c7d6-4bc9-b330-66b29629a2dc.png"  width=100% height=30%> |  Codewars   | <img src="https://user-images.githubusercontent.com/76241195/147094572-962f911f-9ebd-4615-b72f-ff076f4033e8.png"  width=100% height=30%> |   Graduation Hat  | <img src="https://user-images.githubusercontent.com/76241195/147094631-50f94887-db14-4b6a-ab41-938e41be00e4.png" width=100% height=30%>  |
+| Tiktok     | <img src="https://user-images.githubusercontent.com/76241195/147094674-a0b67f90-c62e-4162-93b5-dd313611ff28.png" width=100% height=30%> |  Send   | <img src="https://user-images.githubusercontent.com/76241195/147096133-20e75d9a-0f96-4881-a165-5e7edf1bfe96.png"  width=100% height=30%> | Search    | <img src="https://user-images.githubusercontent.com/76241195/147102645-0d50ba65-5f21-4245-9275-6c3aec7d193c.png" width=100% height=30%>  |
```
Comment (Panquesito7) [inline README.md:112]:
```suggestion
| TikTok     | <img src="https://user-images.githubusercontent.com/76241195/147094674-a0b67f90-c62e-4162-93b5-dd313611ff28.png" width=100% height=30%> |  Send   | <img src="https://user-images.githubusercontent.com/76241195/147096133-20e75d9a-0f96-4881-a165-5e7edf1bfe96.png"  width=100% height=30%> | Search    | <img src="https://user-images.githubusercontent.com/76241195/147102645-0d50ba65-5f21-4245-9275-6c3aec7d193c.png" width=100% height=30%>  |
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — COMMENTED** [review]
The list of icons is becoming long as well as the README.
Either we need to change it from 3 columns to 4 columns or put the Icons in a separate MarkDown File. We can add some regular ones on the README.

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@
   >
   > Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values of the event will count. Maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
  
-</details> 
+ </details>
```
Comment (krishguptadev) [inline README.md:18]:
```suggestion
</details> 
```

**Comment by eddiejaoude** [comment]
Separate markdown file could be a good idea 👍

**Comment by emmalearnscode** [comment]
> The list of icons is becoming long as well as the README.
> Either we need to change it from 3 columns to 4 columns or put the Icons in a separate MarkDown File. We can add some regular ones in the README.

Maybe we can get this pr merged then open a new issue to move them to their own file.

**Comment by Pradumnasaraf** [comment]
> > The list of icons is becoming long as well as the README.
> > Either we need to change it from 3 columns to 4 columns or put the Icons in a separate MarkDown File. We can add some regular ones in the README.
> 
> Maybe we can get this pr merged then open a new issue to move them to their own file.

Yeah, I was also thinking of same.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
My comments are still not resolved. See https://github.com/EddieHubCommunity/LinkFree/pull/844#discussion_r774108323 https://github.com/EddieHubCommunity/LinkFree/pull/844#discussion_r774108158

**Comment by emmalearnscode** [comment]
> My comments are still not resolved. See [#844 (comment)](https://github.com/EddieHubCommunity/LinkFree/pull/844#discussion_r774108323) [#844 (comment)](https://github.com/EddieHubCommunity/LinkFree/pull/844#discussion_r774108158)

Have a little patience @Panquesito7 it’s been less than a day since you requested changes and not everyone has time to code every day. I’m sure @RanjeetNSuthar will make the changes when they have time.

**Review by eddiejaoude — APPROVED** [review]
Thank you @RanjeetNSuthar for the great PR and everyone for the collaboration 👍

**Comment by Panquesito7** [comment]
> Have a little patience @Panquesito7 it’s been less than a day since you requested changes and not everyone has time to code every day. I’m sure @RanjeetNSuthar will make the changes when they have time.

Yeah, I just realized 😓 Sorry for that. Thanks.

**Pull request merged**